CMakeLists.txt

```diff
@@ -10,7 +16,29 @@ include(GNUInstallDirs)
 include(BuildPlatformDetection)
 include(SetProjectDependencies)
 
-set(PYTHON_EXECUTABLE ${CMAKE_INSTALL_PREFIX}/bin/python3)
+# For all OSs:
+if($ENV{PYTHONPATH})
+        set(ENV{PYTHONPATH} ${CMAKE_INSTALL_PREFIX}/lib/python3/dist-packages/:${CMAKE_INSTALL_PREFIX}/lib/python3.5:${CMAKE_INSTALL_PREFIX}/lib/python3.5/site-packages:$ENV{PYTHONPATH})
```

Comment:
Note that these are _not_ carried over to child processes that are called by CMake, so they do not have any effect on ExternalProject for example, which means setting them is rather useless.